pkg/controllers/monitoring/monitoring-stack/components.go

```diff
@@ -172,6 +172,7 @@ func newPrometheus(
 				RemoteWrite:               config.RemoteWrite,
 				ExternalLabels:            config.ExternalLabels,
 				EnableRemoteWriteReceiver: config.EnableRemoteWriteReceiver,
+				ScrapeInterval:            config.ScrapeInterval,
```

Comment:
If we make it a pointer then we need to test for nil.

Reply:
Added a check for nil :)